FAQ.md

```diff
@@ -31,22 +30,34 @@ This is especially problematic for PDB files from X-ray crystallography which ty
 For our purposes here, however, we assume you begin with the coordinates of all atoms present and the full topology of your system.
 
 Given a PDB file of a hypothetical biomolecular system of interest containing a small molecule, there are several routes available to you for treating the small molecule present:
-- Use a cheminformatics toolkit (see above) to infer bond orders
+- Use a cheminformatics toolkit (see below) to infer bond orders
 - Identify your ligand from a database; e.g. if it is in the Protein Data Bank (PDB), it will be present in the [Ligand Expo](http://ligand-expo.rcsb.org) meaning that it has a database entry and code you can use to look up its putative chemical identity
 - Identify your ligand by name or SMILES string (or similar) from the literature or your collaborators
 
+## What about starting from an XYZ file?
+
+XYZ files generally only contain elements and positions, and are therefore similar in content to PDB files. See the above section "What about starting from a PDB file?" for more information.
+
 ## What do you recommend as a starting point?
 
 For application of SMIRNOFF force fields, we recommend that you begin your work with formats which provide the chemical identity of your small molecule (including formal charge and bond order).
 This means we recommend one of the following or equivalent:
-- A `.mol2` file or files for the molecules comprising your system, with correct bond orders and formal charges. (Note: Do NOT generate this from a simulation package or tool which does not have access to bond order information; you may end up with a `.mol2` file, but the bond orders will be incorrect)
+- A `.mol`, `.sdf`, or `.mol2` file or files for the molecules comprising your system, with correct bond orders and formal charges. (Note: Do NOT generate this from a simulation package or tool which does not have access to bond order information; you may end up with a correct-seeming file, but the bond orders will be incorrect)
```

Comment:
This was a particularly glaring omission that I forgot to mention earlier - given that we seem to prefer SDF to MOL2!

Reply:
Yeah, I haven't looked at the FAQ in a while it seems!

Reply:
```suggestion
- A `.sdf`, `.mol`, or `.mol2` file or files for the molecules comprising your system, with correct bond orders and formal charges. (Note: Do NOT generate this from a simulation package or tool which does not have access to bond order information; you may end up with a correct-seeming file, but the bond orders will be incorrect)
```

This is the nittiest nit, but elsewhere in the documentation for this project and others we're very consistent about recommending SDF, so it makes sense to put it first here.